src/test/factories/responses/gitLab/listPrsResponseFactory.ts

```diff
@@ -0,0 +1,6 @@
+import Bitbucket from 'bitbucket';
+import { MergeRequest } from '../../../../services/gitlab/gitlabClient/resources/MergeRequests';
+
+export const gitLabListPRsResponseFactory = (items: MergeRequest[] = []): MergeRequest[] => {
+  return items;
+};
```

Comment:
Shouldn't be there a pagination?

Reply:
As I've discussed test for pagination with you, there should be pagination but I'll add it in another PR, so I don't block this one.